autogpt_platform/backend/backend/util/settings.py

```diff
@@ -117,11 +117,6 @@ class Config(UpdateTrackingModel["Config"], BaseSettings):
         description="The port for agent server daemon to run on",
     )
 
-    database_api_host: str = Field(
```

Comment:
Removing this as it was added in https://github.com/Significant-Gravitas/AutoGPT/commit/6f07d24e9303ec496f12fc281f1d45d32f2f3c0d#diff-9f50c422480771c499000baff849597248109048f6277e0381fe973e494a30eb

but not used for the intention of running services on different hosts